plugins/leemons-plugin-users/frontend/src/pages/private/users/ListUsers/index.js

```diff
@@ -55,13 +58,15 @@ function ListUsers() {
 
   async function listUsers(searchQuery) {
     const query = {};
+
     if (searchQuery ?? store.search) {
+      const search = searchQuery ?? store.search;
```

Comment:
Not sure if the query should run over searchQuery only then the condition is wrong or due to the condition it must use searchQuery ?? store.search

Reply:
We've agreed on defining the search variable outside the if and checking against the variable